### DISCUSSION TITLE
integrating/shortcuts

### DISCUSSION THREAD

**Discussion by giscus** [velopack/velopack.docs#12] [General]
# integrating/shortcuts

By default, during installation Velopack will create a shortcut on the Desktop and in the StartMenuRoot. It will automatically delete any shortcuts it finds when uninstalling the application.

https://docs.velopack.io/integrating/shortcuts

**Comment by yprism** [comment]
1. how to pass arguments when creating a shortcut?
2. how to create shortcut for other files other than main exe?

**Reply by caesay** [reply]
For now, default installer will create a shortcut for the main exe on the Desktop and StartMenuRoot automatically for you. https://github.com/velopack/velopack/issues/46 will make this configurable. Also see https://docs.velopack.io/integrating/shortcuts if you need to create shortcuts for other binaries.

**Comment by Pyritie** [comment]
Please don't enforce creating shortcuts for all installed apps. I used squirrel (and am now migrating to velopack) to install tools that are ran by other apps (such as perforce) that users should never run by themselves, and so they should never be on the user's desktop/start menu.

**Reply by caesay** [reply]
Thanks for the feedback, please see https://github.com/velopack/velopack/issues/46.
